src/voice-pe/about-the-device/status-colors.md

```diff
@@ -6,17 +6,63 @@ zendesk:
   labels: voice pe, about the system, status colors
 ---
 
+{% steps %}
+{% step "The LED closest to the speaker is red." "1 red LED" %}
+{% image "/static/img/voice-pe/status_silent.png" "1 red LED" %}
+{% stepContent %}
+<b>Silent</b><br>If the media player volume is set to 0, the LED closest to the speaker is red, indicating the device cannot produce sound.
```

Comment:
```suggestion

**Silent**

If the media player volume is set to 0, the LED closest to the speaker is red, indicating the device cannot produce sound.
```
When possible, we should use markdown syntax instead of HTML.

Reply:
it didn't work in my preview... Let me give it another shot.